webmunk-ext/webpack.addon.config.dev.js

```diff
@@ -28,6 +28,35 @@ module.exports = mergeWithCustomize({
   mode: 'development',
   devtool: 'inline-source-map',
   devServer: {},
+  module: {
```

Comment:
Can we move this to config.base to avoid code duplication? Also check module configs.